Gems/ROS2/Code/Source/SystemComponents/ROS2CameraSystemComponent.cpp

```diff
@@ -0,0 +1,95 @@
+/*
```

Comment:
Directory "SystemComponents" could be misleading, as it has only some of system components, but not all of them.
Perhaps the new camera SC should go to the Camera folder instead (much like it is with lidar)?